cmd/podman/generate_kube.go

```diff
@@ -25,7 +25,9 @@ var (
 			containerKubeCommand.GlobalFlags = MainGlobalOpts
 			return generateKubeYAMLCmd(&containerKubeCommand)
 		},
-		Example: "CONTAINER|POD-NAME",
+		Example: `podman generate kube ctrID
+  podman generate kube podID
+  podman generate kube --service podID`,
```

Comment:
@baude Didn't we remove `--service`?

Reply:
It's still in the man page and looks to be in the code too:
```
        if c.Bool("service") {
                output = append(output, []byte("---\n")...)
                output = append(output, marshalledService...)
        }
```